go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/adlio/schema v1.3.0
 	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/celestiaorg/nmt v0.8.0
+	github.com/celestiaorg/nmt v0.9.1-0.20220529205217-a1a0a2f56fce
```

Comment:
```suggestion
	github.com/celestiaorg/nmt v0.9.0
```

Reply:
We do still have to change this, but to v0.10.0 after we merge https://github.com/celestiaorg/nmt/pull/58

I was incorrect when I said that v0.9.0 would be sufficient, we also have to be able to verify proofs that span more than one nmt leaf.

Reply:
Oh, I missed that one. Will have a 👀

Reply:
approooooved!